### PR TITLE
Fix wrong frame layout stroke attrs using

### PR DESCRIPTION
**Pull request by luongvo**

### CODE REVIEW

File: carbon/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -482,15 +482,16 @@
         <attr name="carbon_insetBottom"/>
         <attr name="carbon_insetColor"/>
 
+        <attr name="carbon_stroke" />
```
Comment (ZieIony) [inline carbon/src/main/res/values/attrs.xml]: pretty much all views have stroke attrs after cornerRadius. I don't think that this refactor is neccessary

**Comment by ZieIony** [comment]
Nice finding! Could you please revert the refactor commit? I would like to merge just the fix

File: carbon/src/main/res/values/attrs.xml (hunk 1 above)
Comment (luongvo) [reply carbon/src/main/res/values/attrs.xml]: @ZieIony got your idea

**Comment by luongvo** [comment]
@ZieIony I just reverted the refactor commit

**Review by ZieIony — APPROVED** [review]

**Pull request merged**